packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -654,6 +656,17 @@ export class DataStores implements IDisposable {
         }
     }
 
+    /**
+     * Assumes that the datastore exists and throws an error if it doesn't exist
```

Comment:
Maybe modify the comment to reflect how the callers should use it, i.e., something like call this to get a data store context from an id. It the data store doesn't exist, it will throw an error.

Reply:
Having said that, I usually prefer this kind of code to be inline in the caller as the validation happens there itself.

Reply:
I think I may have spent time overengineering the solution as I wanted the same assert code to be used for very similar use cases.

Reply:
This allows me to avoid multiple of the same assert error codes as well

Reply:
Moved inline